tests/IntegrationTestCase.php

```diff
@@ -95,7 +95,11 @@ public static function tearDownAfterClass(): void
      */
     protected static function cleanDir($dir)
     {
-        PH::runOk(['if [ -d @DIR ]; then rm -rf @DIR ; fi', 'DIR' => $dir]);
+        if (PHP_OS_FAMILY === 'Windows') {
```

Comment:
Do you want to change to `Platform::isWindows()` ?

Reply:
Oh, I hadn't noticed `PHP_OS_FAMILY` before. Looks like it was added to stdlib in 7.2. Either seems fine to me.